cpp/tests/wrappers/bools_test.cpp

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/wrappers/bool.hpp>
+
+#include <tests/utilities/cudf_gtest.hpp>
+
+#include <random>
+
+struct Bool8Test : public ::testing::Test {
+
+  Bool8Test()
+      : dist{std::numeric_limits<uint8_t>::lowest(),
+             std::numeric_limits<uint8_t>::max()} {
+
+    // Use constant seed for deterministic results
+    rng.seed(0);
+  }
+
+  std::mt19937 rng;
+  std::uniform_int_distribution<uint8_t> dist;
+
+  uint8_t rand(){
+      return dist(rng);
+  }
+};
+
+template <typename SourceType>
+struct Bool8CtorTest : public ::testing::Test {
```

Comment:
Always use the cudf base fixture instead of the gtest one directly.
```suggestion
struct Bool8CtorTest : public cudf::test::BaseFixture {
```

Reply:
Fixed